core/src/com/unciv/logic/simulation/MutableInt.kt

```diff
@@ -1,11 +1,13 @@
 package com.unciv.logic.simulation
 
+import com.unciv.models.translations.tr
+
 class MutableInt(var value: Int = 0) {
     fun inc() { ++value }
     fun get(): Int { return value }
     fun set(newValue: Int) { value = newValue }
 
     override fun toString(): String {
-        return value.toString()
+        return value.tr()
```

Comment:
This is only used in simulation where we don't use translations, I think this is unnecessary
It doesn't hurt though, so not a blocker